packages/mui-material/src/InputLabel/InputLabel.test.js

```diff
@@ -118,6 +119,40 @@ describe('<InputLabel />', () => {
         setProps({ shrink: true });
         expect(getByTestId('root')).to.have.class(classes.shrink);
       });
+
+      it('provides ownerState.focused in styleOverrides', () => {
+        const colorRed = 'rgb(255, 0, 0)';
+
+        const theme = createTheme({
+          components: {
+            MuiInputLabel: {
+              styleOverrides: {
+                root: (props) => {
+                  return {
+                    ...(props.ownerState.focused === true && {
+                      [`&.${classes.focused}`]: {
+                        color: colorRed,
+                      },
+                    }),
```

Comment:
`color` isn't the best property to use here as it has a specificity conflict with `FormLabel`: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/FormLabel/FormLabel.js#L47-L49